Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -201,7 +201,7 @@ private void OnBuckleAttempt(Entity<CuffableComponent> ent, EntityUid? user, ref
             if (cancelled || user != ent.Owner)
                 return;
 
-            if (!TryComp<HandsComponent>(ent, out var hands) || ent.Comp.CuffedHandCount != hands.Count)
+            if (!TryComp<HandsComponent>(ent, out var hands) || ent.Comp.CuffedHandCount <= hands.Count)
                 return;
 
```

Comment:
Isn't this going to always return? When would cuffedhandcount ever exceed their handscount?

Reply:
monkeys

Reply:
changed to less than. Didn't notice it because being cuffed also removes the verb, so it only matters for ai.